android/src/main/java/com/reactnativestripesdk/PaymentMethodCreateParamsFactory.kt

```diff
@@ -439,6 +454,45 @@ class PaymentMethodCreateParamsFactory(
         clientSecret = clientSecret,
       )
   }
+
+  @Throws(PaymentMethodCreateParamsException::class)
+  private fun createUSBankAccountPaymentConfirmParams(): ConfirmPaymentIntentParams {
+    if (billingDetailsParams?.name.isNullOrBlank()) {
+      throw PaymentMethodCreateParamsException("When creating a US bank account payment method, you must provide the following billing details: name")
+    }
+
+    return ConfirmPaymentIntentParams.createWithPaymentMethodCreateParams(
+      paymentMethodCreateParams = createUSBankAccountParams(params),
+      clientSecret,
+      setupFutureUsage = mapToPaymentIntentFutureUsage(getValOr(params, "setupFutureUsage"))
+    )
+  }
+
+  @Throws(PaymentMethodCreateParamsException::class)
+  private fun createUSBankAccountParams(params: ReadableMap): PaymentMethodCreateParams {
```

Comment:
Technically when creating a `us_bank_account` payment method type, we only require the `type` and `name`.